Update linux.md (#1597)
cloudflare/cloudflare-docs: products/warp-client/src/content/setting-up/linux.md

# Linux desktop client

## Known Issues

> Thank you for using the intial release of the Linux client for WARP! Here is a list of known issues. Please check back frequently as we keep this updated.

Reviewer: ```suggestion
Thank you for using the initial release of the Linux client for WARP! In this section we will call out known issues. Please check back frequently as we keep this list updated.
```

> * Cloudflare for Teams is not yet fully supported

Reviewer: Do we need to call out what is and what isn't supported? The word `fully` makes me wonder

1. Connect `warp-cli connect`
1. Run `curl https://www.cloudflare.com/cdn-cgi/trace/` and verify that `warp=on`

### Always stay connected

> If you want to always stay connected to WARP you must call `enable-always-on`. This is the cli equivilent to switching the toggle switch to the on position in our GUI apps.

Reviewer: ```suggestion
If you want WARP to be constantly enabled, call `enable-always-on`. This is the CLI equivalent to switching the toggle switch to the on position in our UI.
```

> 1. Run `warp-cli enable-always-on`

Reviewer: ```suggestion
The command you need to run is:

```sh
$ warp-cli enable-always-on
```
```
^ please check whether the `$` is correct! if not feel free to adjust
